templates/regulation.yaml

```diff
@@ -0,0 +1,17 @@
+genome_id: <genome_id>
+category:
+  track_category_id: regulatory-features
+  label: Regulation
+  type: Regulation
+trigger:
+  - track
+  - regulation
+type: regular
+label: Regulatory annotation
+display_order: 890
+on_by_default: true
+description: |-
+  Promoters, enhancers and open chromatin regions are available for all species. 
+  Transcription factor binding and CTCF binding sites are only available for human and mouse.
```

Comment:
Do you know why we keep writing this in the description? I don't think we are even showing this anywhere in the genome browser... Also, I can't remember whether we hide the "TF binding" and the CTCF colour blocks on genomes other than human and mouse. Probably not?

<img width="1204" alt="image" src="https://github.com/user-attachments/assets/0ebde42b-b1db-4496-a300-2c7466a5caa2">

Reply:
The regulation track description field comes from [here](https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-1969). But we also have feature-specific descriptions hardcoded in the client, don't we?

Reply:
> Where do these feature-specific descriptions come from, is it hardcoded in the client?

Yes, they are hard-coded on the client. These feature-specific descriptions look visually the same as tracks, but aren't tracks, and thus can't be using the track api.

https://github.com/Ensembl/ensembl-client/blob/80360adb0a6117f77753ebb0e2a5457c7ec29b36/src/content/app/genome-browser/constants/regulationLegend.ts#L51

I also found where we are showing the description of the regulation track:

![image](https://github.com/user-attachments/assets/d4ffa448-95c8-4d72-9a26-141655bdd71b)

It is still a bit odd that we write about "human and mouse" regardless of the genome that we are showing 🤷‍♂️